ggml-alloc.c

```diff
@@ -775,7 +778,7 @@ ggml_backend_buffer_t ggml_backend_alloc_ctx_tensors_from_buft(struct ggml_conte
     }
 
     if (nbytes == 0) {
-        fprintf(stderr, "%s: no tensors to allocate\n", __func__);
+        //fprintf(stderr, "%s: no tensors to allocate\n", __func__);
```

Comment:
When fully offloading the KV cache with CUDA, all the tensors are already allocated and there is nothing to allocate. This is not an error or a problem, so I removed the print to avoid confusion.

Reply:
Then I think you should remove the printf (not just comment it) and add a comment explaining why this is normal.